<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrikCoefTypeEditorDialog</class>
 <widget class="QDialog" name="PrikCoefTypeEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поправочные коэффициенты</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblPeriodFrom">
       <property name="text">
        <string>с</string>
       </property>
       <property name="buddy">
        <cstring>edtBegDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBegDate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPeriodTo">
       <property name="text">
        <string>по</string>
       </property>
       <property name="buddy">
        <cstring>edtEndDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CInsurerComboBox" name="cmbInsurer">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QComboBox" name="cmbCoefType">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>К2 | Поправочный коэффициент стоимости пролеченных больных</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="CInDocTableView" name="tblPrikCoefItem">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblInsurer">
     <property name="text">
      <string>Страховая компания</string>
     </property>
     <property name="buddy">
      <cstring>cmbInsurer</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPeriod">
     <property name="text">
      <string>Период</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCoefType">
     <property name="text">
      <string>Тип коэффициента</string>
     </property>
     <property name="buddy">
      <cstring>cmbCoefType</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header>library/InDocTable.h</header>
  </customwidget>
  <customwidget>
   <class>CInsurerComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgComboBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrikCoefTypeEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrikCoefTypeEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
